hack/dailybuild/dailybuild.go

```diff
@@ -244,15 +243,15 @@ func postDiscussion(token string, builds *Builds, releaseNotes string) error {
 	} else {
 		bodyTemplate += "%s" // this effectively becomes a noop because it will be empty
 	}
+
 	jsonTemplate := "{\n" +
-		"\"query\": \"mutation {   createDiscussion(input: {repositoryId: \\\"MDEwOlJlcG9zaXRvcnkzMDM4MDIzMzI\\\", categoryId: \\\"DIC_kwDOEhun3M4CA9pd\\\", body: \\\"%s\\\", title: \\\"%s\\\"}) {     discussion {       id     }   } }\"\n" +
+		"\"query\": \"mutation {   addDiscussionComment(input: {discussionId: \\\"D_kwDOEhun3M4AO9Ut\\\", body: \\\"%s\\\"}) { comment {       id     }   } }\"\n" +
```

Comment:
Looks good according to https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions#adddiscussioncomment